scripts/objc-test.sh

```diff
@@ -26,7 +26,16 @@ trap cleanup EXIT
 
 node ./packager/packager.js --nonPersistent &
 SERVER_PID=$!
+
+xctool \
+  -project Examples/2048/2048.xcodeproj \
+  -scheme 2048 \
+  -sdk iphonesimulator \
+  test
+
 xctool \
   -project Examples/UIExplorer/UIExplorer.xcodeproj \
-  -scheme UIExplorer -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 5,OS=8.3' \
+  -scheme UIExplorer \
+  -sdk iphonesimulator \
+  -destination 'platform=iOS Simulator,name=iPhone 5' \
```

Comment:
OS=8.3 didn't work on my machine. Let's try to remove it and see what travis says ...